lingui.config.js

```diff
@@ -0,0 +1,10 @@
+/** @type {import('@lingui/conf').LinguiConfig} */
+module.exports = {
+	locales: ["en", "fr"],
+	sourceLocale: "en",
+	catalogs: [{
+	   path: "src/locales/{locale}/messages",
```

Comment:
This is where messages are located, this can also be changed to have messages to be next to each component, not within its own locales and just named the locale.po, and many more choices. I do believe that most translators would rather have one big file to translate since it is easier to not miss anything. This however causes the file to grow larger and larger.

I do see some benefits to separating them either into `{locale}/{component}/messages` or `src/components/{component}/locales/{locale}/messages` but I'm not sure if they are worth it.

On one side having a giant messages allows all translations to stay in one spot, not worry about having to track each translation file down, multiple files can rely on only one string to be changed to change the displayed string in multiple places.

On the other side has smaller messages, no issues where two matching strings should have two different translations because of context, and keeps component information together.

Reply:
I agree with your instinct here - let's keep each locale in one file. We can always pivot later if it becomes necessary.